api_v3/lib/types/partner/KalturaPartner.php

```diff
@@ -440,10 +440,10 @@ class KalturaPartner extends KalturaObject implements IFilterable
 	public $numPrevPassToKeep;
 	
 	/**
-	 * @var KalturaTwoFactorAuthenticationMode
+	 * @var bool
 	 * @readonly
 	 */
-	public $twoFactorAuthenticationMode;
+	public $purifyImageContent;
```

Comment:
purifyImageContent property should have been removed, not twoFactorAuthenticationMode